packages/ds-healthcare-gov/src/styles/components/_Footer.scss

```diff
@@ -43,8 +43,9 @@
     padding: 0;
     text-decoration: none;
 
+    &:hover,
     &:focus {
-      @include mixins.focus-styles;
+      color: var(--color-primary-darker);
```

Comment:
Why was this change made to focus? What was there before wasn't perfect either, but this is what it looks like now:

![focus](https://user-images.githubusercontent.com/7595652/225134491-ea4aa187-ef0f-46b1-a8bd-f017087a9e68.gif)

Reply:
Oh shoot didn't notice the focus state, I'll make that only on hover. The hover state previously is grey and should be blue.

Reply:
Updated a couple things here to resolve this
![image](https://user-images.githubusercontent.com/783820/225147259-9e4d1dd4-868d-4a1c-8ea4-5dee464b030a.png)

Reply:
Nice, thank you 👍